modules/apps/commerce/commerce-channel-web/test/js/__mock__/product.ts

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
Do this

https://github.com/is-solutions-delivery/liferay-portal/pull/6165#discussion_r1932620226